src/MessageList/AttachmentButton.vala

```diff
@@ -22,6 +22,16 @@ public class AttachmentButton : Gtk.FlowBoxChild {
     public Camel.MimePart mime_part { get; construct; }
     public unowned GLib.Cancellable loading_cancellable { get; construct; }
 
+    private const string ACTION_GROUP_PREFIX = "attachmentbutton";
+    private const string ACTION_PREFIX = ACTION_GROUP_PREFIX + ".";
+    private const string ACTION_OPEN = "open";
+    private const string ACTION_SAVE_AS = "save-as";
+
+    private const ActionEntry[] ACTION_ENTRIES = {
+        {ACTION_OPEN, on_open},
+        {ACTION_SAVE_AS, on_save_as}
+    };
```

Comment:
It would probably be simpler here to just use [`SimpleAction`](https://valadoc.org/gio-2.0/GLib.SimpleAction.html) instead of making an array of action entries. I'm not sure who started this trend of using action entries like this tbh :sweat_smile:

Reply:
Like this or is there an even easier way?